ceph/cephfs/README.md

```diff
@@ -48,6 +48,12 @@ kubectl create -f claim.yaml
 kubectl create -f test-pod.yaml
 ```
 
+* Deploy
```

Comment:
it should be this *or* "Start CephFS provisioner" above I think. Else a user going step by step will do both and have 2 instances running & be very confused.